src/main/java/com/dev/cinema/annotation/FieldsValueMatch.java

```diff
@@ -0,0 +1,31 @@
+package com.dev.cinema.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Documented
+@Constraint(validatedBy = FieldsMatchValidator.class)
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldsValueMatch {
+    String message() default "Fields are not matching";
+
+    Class<?>[] groups() default {};
+
+    Class<? extends Payload>[] payload() default {};
+
+    String field();
+
+    String fieldMatch();
+
+    @Target({ElementType.TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface List {
+        FieldsValueMatch[] value();
+    }
```

Comment:
you don't need this block (from line 26) - you have only one pair (password - passwordRepeat). You need it when you have several such pairs (eg email - repeatEmail etc)

Reply:
done